impeller/entity/contents/texture_contents.cc

```diff
@@ -141,9 +150,16 @@ bool TextureContents::Render(const ContentContext& renderer,
   frag_info.alpha = opacity_;
 
   Command cmd;
-  cmd.label = "TextureFill";
-  cmd.pipeline =
-      renderer.GetTexturePipeline(OptionsFromPassAndEntity(pass, entity));
+  cmd.label = "Texture Fill";
+  if (!label_.empty()) {
+    cmd.label += ": " + label_;
+  }
+
+  auto pipeline_options = OptionsFromPassAndEntity(pass, entity);
+  if (!stencil_enabled_) {
+    pipeline_options.stencil_compare = CompareFunction::kAlways;
```

Comment:
I wonder if stencil read/write masks are part of the PSO description. If not, we can get away with with using those instead of creating PSO variants.

Reply:
> I always find the compare function confusion to reason about

I think the inequality direction is what usually gets me.

> I thought kAlways is always pass instead of fail and wasn't sure how all this worked.

Oh uhh, I think it is? That's how I was intending to use it anyway: "Disabling" the stencil = always passing the check.

> I wonder if stencil read/write masks are part of the PSO description.

If I'm reading the Metal docs right, it looks like it is unfortunately. Vulkan seems to have pipeline commands to set the stencil masks, though.